ecs-cli/modules/cli/configure/migrate.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package configure
+
+import (
+	"html/template"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
+)
+
+func hideCreds(cliConfig *config.CLIConfig) {
+	if cliConfig.AWSSecretKey != "" {
+		cliConfig.AWSSecretKey = "********************" + cliConfig.AWSSecretKey[len(cliConfig.AWSSecretKey)-2:]
```

Comment:
I don't think it matters what length the "*********" sequence is, I think customers will be able to figure it out. But, since you provided the code, I'll use it ;)